src/migration/1687349045215-DisallowHandle.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DisallowHandle1687349045215 implements MigrationInterface {
+  name = 'DisallowHandle1687349045215';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "disallow_handle" ("value" character varying NOT NULL, CONSTRAINT "PK_8c84e48365905231826700dd8b4" PRIMARY KEY ("value"))`,
+    );
+    await queryRunner.query(
```

Comment:
Probably this index was created due to the `{unique: true}`. We don't need it so let's drop it from the entity and regenerate the migration. Sorry for the trouble! I'll not block for this